### PR TITLE
MVS-481 adjust the arrangement of fields on the enter your personal info pages

### PR DESCRIPTION
**Pull request by dmcook5208**
## :flipper: [JIRA Sprint Card]
https://mass-immunization-system.atlassian.net/browse/MVS-481

## :newspaper: [Work Description]
Updated the layout of the SinglePatientPersonalInfo, HouseholdPersonalInfo_1, and HouseholdPersonalInfo_n pages per the recommendations captured in the card (and reviewed prototypes with David, Peter, Teri).

## :vertical_traffic_light: [Testing/QA Notes]
Build and load the application
Verify that the single patient personal info page is formatted as expected per the acceptance criteria of the card
Verify that the household patient registration (for patient #1) is formatted as expected per the acceptance criteria of the card
Verify that the household patient registration (for patients #2-20) is similar to the format of the single patient and household patient #1 pages.

### CODE REVIEW

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 1]
```diff
@@ -1,60 +1,57 @@
 <template>
-	<v-container fluid>
-	<v-row align="center" justify="left">
-		<!-- Last name -->
-		<v-col class="d-flex" cols="4" sm="4">
+	<v-container>
+    <v-row align="center" justify="start">
+
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:4]: Why is there an extra space here? We should probably for consistency and readability.

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 2]
```diff
@@ -157,29 +143,29 @@ import EventBus from "../eventBus";
 export default {
   data() {
     return {
-      genderID: ["Male", "Female", "Other", "Decline to answer"],
-      race: [
+      genderIDOptions: ["Male", "Female", "Other", "Decline to answer"],
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:146]: Do we typically write this as `genderID` or does it change throughout our code base? I think `genderIdOptions` is better.

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 3]
```diff
@@ -157,29 +143,29 @@ import EventBus from "../eventBus";
...
+      raceOptions: [
         "Black or African American",
         "White",
         "Asian",
         "American Indian or Alask a Native",
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:151]: Let's fix the awk spacing in "Alaska"

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 4]
```diff
@@ -65,88 +62,77 @@
...
       </v-col>
-      <v-spacer></v-spacer>
+    </v-row>
+    <v-row align="left" justify="space-between">
+      <v-col cols="5">
+
+        <!-- Current Photo -->
+        <v-file-input
+          accept="image/png, image/jpeg, image/bmp"
+          :rules="[(v) => (v ? v.size : 0) < 2097152 || 'Image size should be less than 2 MB!']"
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:129]: Can you add in a comment clarifying this logic

**Review by david-fraley — CHANGES_REQUESTED** [review]

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 1 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:4]: Removed extra spaces throughout (same change applied to all 3 files)

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 2 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:146]: Changed all references (in all 3 files) to genderIdOptions

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 3 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:151]: fixed in all 3 files

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 4 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:129]: Added comment (same comment to all 3 files)

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 5]
```diff
@@ -1,60 +1,52 @@
...
+    <v-row align="center" justify="start">
+      <!-- Last name -->
+      <v-col cols="12" sm="6" md="6" lg="4">
         <v-text-field 
-          id="lastName" 
-		required
+          required
           :rules="[v => !!v || 'Last name field is required']"
-          v-model="householdFamilyName"
+          v-model="familyName"
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:9]: Why are you changing this field name?

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue [hunk 6]
```diff
@@ -65,121 +57,106 @@
         >
           <template v-slot:activator="{ on }">
             <v-text-field
-				v-model="householdDate"
-				:rules="birthdateRules"
+              v-model="birthDate"
```
Comment (david-fraley) [inline PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:60]: This one as well?

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 4 above)
Comment (david-fraley) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:121]: This change was just introduced due to a merge conflict or something?

**Review by david-fraley — CHANGES_REQUESTED** [review]
Gotta merge Dev in again since this branch is 18 commits behind

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 4 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:121]: Yes - in Tom's merge that introduced these changes, the "current photo" was above race and ethnicity options.  I had to cut and paste it to the new position.

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 5 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:9]: By standardizing the names for the common elements we have between SinglePatientPersonalInfo, HouseholdPersonalInfo_1, and HouseholdPersonalInfo_n, we:
- get one step closer to eventually consolidating those 3 components into 1 (for efficiency)
- make it easier for someone to work on all 3 files (by calling the same components by the same name in all 3 places)
- make the names a bit less verbose (we really don't need to have "household" in front of every variable name

File: PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue (hunk 6 above)
Comment (dmcook5208) [reply PatientRegistrationApp/src/components/HouseholdPersonalInfo_1.vue:60]: "birthDate" makes more sense than "householdDate" for the Date of Birth field

**Comment by dmcook5208** [comment]
I replied to your comments!  Please let me know if there is anything else

**Review by david-fraley — APPROVED** [review]
Looks good

**Pull request merged**